bin/sl-run.js

```diff
@@ -18,6 +18,30 @@ var agentOptions = {
   logger: config.logger,
 };
 
+var USE_PROBES = true;
+
+if (config.enableTracing) {
+  var tracer = require('../lib/tracer');
+  var traceObject = require('../lib/trace-object');
```

Comment:
why don't you set the tracer options in https://github.com/strongloop/strong-supervisor/blob/master/lib/trace-object.js#L6, with the other tracer options? splitting them across two places seems confusing, and its a lot of code to put in the middle of sl-run.js